README.md

```diff
@@ -980,37 +983,74 @@ CredentialAccessBoundary credentialAccessBoundary =
         CredentialAccessBoundary.newBuilder().addRule(rule).build();
 ```
 
+#### Common Usage Pattern
+
 The common pattern of usage is to have a token broker with elevated access generate these downscoped
 credentials from higher access source credentials and pass the downscoped short-lived access tokens
 to a token consumer via some secure authenticated channel for limited access to Google Cloud Storage
 resources.
 
-Using the CredentialAccessBoundary created above in the Token Broker:
+#### Generating Downscoped Tokens
+
+There are two ways to generate downscoped tokens using a
+CredentialAccessBoundary:
```

Comment:
nit: If possible, could there be like a table or a pro/ cons comparison between the two that explicitly spells out what any considerations/ impacts of one vs the other?

Also, is there a need for something like a migration guide? To help users potentially migrate from server-side to client-side?

Reply:
Also, is there an general recommendation that that your team would provide between the two. I know it's possible that may not be a recommendation.

i.e. Prefer client-side unless ... X,Y,Z blocker?

Reply:
Checked with the team, we don't have a general recommendation, or a migration guide. Depending on their use case, and whether or not they need many unique downscoped tokens or they can re-use existing ones, they can decide between the two optiosn.